index.js

```diff
@@ -0,0 +1,62 @@
+const express = require('express');
```

Comment:
I also recommend splitting these one liners into more organized sections. Like a section for imports, a section for implementing imports, a section for settings, separated by a new line.